go_api_server/main.go

```
@@ -20,9 +21,30 @@ func main() {
+			Filters: []database.Filter{
+				{
+					Operator: "=",
+					Field:    "bank_id",
+					Value:    []any{1},
+				},
+			},
+		},
+		Fields: []string{"BankID"},
+	}
+
+	data, err := database.ExecuteSelectQuery(query)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)